deploy/operator.yaml

```diff
@@ -16,30 +16,32 @@ spec:
   replicas: 1
   selector:
     matchLabels:
-      name: che-operator
+      app: che-operator
   template:
     metadata:
       labels:
-        name: che-operator
+        app: che-operator
     spec:
-      serviceAccountName: che-operator
       containers:
         - name: che-operator
-          image: quay.io/eclipse-che/che-operator:7.0.0-beta-5.0
```

Comment:
shouldn't we put nightly here as tag and changing the value only when releasing (ideally in a release branch)?

Reply:
+1, great idea (as long as we remember to set it to a specific version after branching for release). :D

Reply:
Then we have to switch to the `Always` pull policy I assume.

Reply:
Done